.infra/package.json

```
@@ -1,12 +1,19 @@
 {
   "name": "api",
+  "scripts": {
+    "prepare": "corepack enable || true"
+  },
   "devDependencies": {
-    "@types/node": "20.12.x"
+    "@types/node": "22.13.x"
   },
   "dependencies": {
     "@dailydotdev/pulumi-common": "^2.4.6",
```

```suggestion
    "@dailydotdev/pulumi-common": "^2.5.0",
```

Released a version that's built with v22